source/_integrations/http.markdown

```diff
@@ -104,6 +111,31 @@ http:
 
 The [Set up encryption using Let's Encrypt](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) blog post gives you details about the encryption of your traffic using free certificates from [Let's Encrypt](https://letsencrypt.org/).
 
+## Multiple Servers (TLS/SSL Configuration per port)
+
+For most use cases, configure the first server with the most secure settings when using multiple servers, as it will be used by default when determining the external URL.
+
+The sample below will listen on multiple ports, which can be helpful for devices that use webhooks that do not work with SSL or use an older TLS/SSL standard.
```

Comment:
```suggestion
The example below shows a configuration that allows listening on multiple ports. This can be helpful for devices that use webhooks that do not work with SSL or use an older TLS/SSL standard.
```